packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.itemPlugin.ts

```diff
@@ -34,6 +36,9 @@ export const useTreeViewItemsReorderingItemPlugin: TreeViewItemPlugin = ({ props
     itemId,
   );
   const isValidTarget = useSelector(store, selectorItemsReorderingIsValidTarget, itemId);
+  const draggedItemId = useSelector(store, selectorDraggedItem);
+
+  const isBeingEdited = useSelector(store, selectorIsItemBeingEdited, draggedItemId as string);
```

Comment:
```suggestion
If `draggedItemId` can be null, you can do:


  const isBeingEdited = useSelector(
    store, 
    state => draggedItemId == nul ? false : selectorIsItemBeingEdited(draggedItemId), 
  );
```

Or you can update `selectorIsItemBeingEdited` to support `null`, which would make sense to me as well.